queries.sql

```diff
@@ -43,3 +43,96 @@ FROM animals
 WHERE weight_kg BETWEEN 10.4 AND 17.3;
 
 
+-- Inside a transaction update the animals table by setting the species column to unspecified. Verify that change was made. 
+BEGIN;
+UPDATE animals
+SET species = 'unspecified';
+
+
+-- Verify that change was made. Then roll back the change and verify that the species columns went back to the state before the transaction.
+SELECT * FROM animals; -- species column = unspecified
+ROLLBACK;
```

Comment:
Well! as the previous reviewer mentioned you worked on this comment. You updated screensoot of the verification. but you must also update the transition in the `queries.sql` file.  You have to add the SELECT query in the `query.sql` file. Please consider updating your transection so that work same way as your screenshot. 👍 
```suggestion
BEGIN;
UPDATE animals
SET species = 'unspecified';


-- Verify that change was made. Then roll back the change and verify that the species columns went back to the state before the transaction.
SELECT * FROM animals; -- species column = unspecified
ROLLBACK;
SELECT * FROM animals; 
```